src/overlaybd/zfile/zfile.cpp

```diff
@@ -674,7 +706,9 @@ bool load_jump_table(IFile *file, CompressionFile::HeaderTrailer *pheader_traile
     if (!pht->verify_magic() || !pht->is_header()) {
         LOG_ERROR_RETURN(0, false, "header magic/type don't match");
     }
-
+    if (pht->is_valid() == false) {
+        LOG_ERROR_RETURN(0, false, "digest verification failed.");
```

Comment:
Just to confirm, will this trigger eviction?

Reply:
it will be evict when build_jump_table failed